docs/mobile-apps/faq.md

```diff
@@ -61,6 +61,16 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
 For real devices on the public cloud, the limit is 6 hours for Appium tests and 1 hour for Espresso and XCUITest; the same applies to private cloud devices.
 The limits for public cloud devices ensure that there are enough devices available for you, whereas the limits for private cloud devices ensure the longevity of your devices.
 
+#### **Is there a difference in devices I can use as a free trial and with a subscription when live testing?**
+
+For free trial accounts, the only available devices you will see are denoted with a green corner flag that states trial and they will be available to you. If you are subscribed, you will have access to a vast pool of devices which exceeds 300 models and behind those models will be different versions of OS available to you. 
+
+Example: iPhone 12
```

Comment:
I am not sure what this example is telling me in regards to the question.

Reply:
There were a list of OS versions under line 68. Something like: 
Example: iPhone 12
iPhone 12 - iOS 15
iPhone 12 - iOS 16
iPhone 12 - iOS 16.1 
and so on, the example should demonstrate that there isn't only 1 iPhone/device, but multiple running multiple versions of OS

Reply:
@saucechaffe, I updated the example.